apps/dapp-console-api/src/models/challenges.ts

```diff
@@ -27,6 +34,7 @@ export const challenges = pgTable(
     entityId: uuid('entity_id')
       .references(() => entities.id)
       .notNull(),
+    chainId: integer('chain_id').notNull(),
```

Comment:
this will make it easier to do signature verification of non-EOA wallets, since a chain id must be specified in order to fetch the correct public client for signature verification.